PipelinesTasks/PRMetrics/buildAndReleaseTask/invokers/azureReposInvoker.ts

```diff
@@ -44,21 +44,21 @@ export default class AzureReposInvoker {
     }
 
     /**
-      * Gets the current iteration id of the pull request from the devops api.
-      * Returns undefined if no iteration exists.
+      * Gets the current iteration of the pull request.
+      * @returns A promise containing the current iteration of the pull request.
       */
-    public async getCurrentIterationId (): Promise<number | undefined> {
-      this.taskLibWrapper.debug('* AzureReposInvoker.getCurrentIterationId()')
+    public async getCurrentIteration (): Promise<number> {
```

Comment:
Why are we renaming this? We are only returning the Id.

Reply:
Oh, we need this to return the iteration number, not the ID